core/src/test/java/org/apache/calcite/test/JdbcAdapterTest.java

```diff
@@ -1355,6 +1355,27 @@ private LockWrapper exclusiveCleanDb(Connection c) throws SQLException {
         .runs();
   }
 
+  /**
+   * Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6995">[CALCITE-6995]
+   * StarRocks/Doris should support full join</a>.*/
```

Comment:
We should be consistent with the others. `</a>. */`

Reply:
Thanks for remidner, had fixed it.